src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -610,6 +742,73 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
     this._hideCurrentEditorChangeDialog();
   }
 
+  _clearSSHKeypairText() {
+    this.shadowRoot.querySelector('#ssh-public-key').value = '';
+    this.shadowRoot.querySelector('#ssh-private-key').value = '';
+
+    // if alert dialog is opened, then hide.
+    if (this.shadowRoot.querySelector('#clear-ssh-keypair-dialog').open) {
+      this._hideSSHKeypairClearDialog();
+    }
+  }
+
+  /**
+   * Toggle readonly input in SSH keypair.
+   */
+  _toggleSSHKeypairInput() {
+    const selected: string = this.shadowRoot.querySelector('#keypair-generation-dropdown').value;
```

Comment:
Here as well. For me, it does not feel good to compare elements' text description.